README.md

```diff
@@ -31,7 +31,7 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
-$ cmake ..
+$ cmake -DCMAKE_PREFIX_PATH=/opt/ros/indigo ..
```

Comment:
I'd rather add a line to the instructions that source `/opt/ros/indigo/setup.bash`, which is the typical way that people use Catkin workspaces. The setup file sets `CMAKE_PREFIX_PATH` as well as potentially a bunch of other variables in the environment hooks. This _may_ work, but it makes me nervous that the build may break in an unpredictable way if it assumes those environment hooks have run.

@psigen Thoughts? Is this an empty fear?